doc/lua_api.txt

```diff
@@ -1673,17 +1673,37 @@ these formats.
 ### Serialized
 
 This is called "stackstring" or "itemstring". It is a simple string with
-1-3 components: the full item identifier, an optional amount and an optional
-wear value. Syntax:
+1-4 components:
 
-    <identifier> [<amount>[ <wear>]]
+1. Full item identifier ("item name")
+2. Optional amount
+3. Optional wear value
+4. Optional item metadata
+
+Syntax:
+
+    <identifier> [<amount>[ <wear>[ <metadata>]]]
 
 Examples:
 
 * `"default:apple"`: 1 apple
 * `"default:dirt 5"`: 5 dirt
 * `"default:pick_stone"`: a new stone pickaxe
 * `"default:pick_wood 1 21323"`: a wooden pickaxe, ca. 1/3 worn out
+* `[[default:pick_wood 1 21323 "\u0001description\u0002My worn out pick\u0003"]]`:
```

Comment:
What would wear do here for a non-tool item? (Maybe add an example for that?)

Reply:
> What would wear do here for a non-tool item? (Maybe add an example for that?)

Don't do it. It will just be hidden.

Reply:
So what value should be used for wear in the 4-part format if it does not make sense? Zero?

Reply:
Yes

Reply:
It may seem obvious, but I think it should be documented.

Reply:
Added an example.